app/admin/news_article.rb

```diff
@@ -7,7 +7,7 @@
   decorate_with NewsArticleDecorator
 
   permit_params :title, :content, :publication_date, :article_type,
-                :created_by_id, :updated_by_id
+                :created_by_id, :updated_by_id, :thumbnail, :image
```

Comment:
Do we need different thumbnail and different image?

Reply:
Or this should be done by active storage image manipulation plugin

Reply:
true, I'll fix that up